### PR TITLE
51 improve performance drop

### LINKED ISSUE #51: Improve performance drop due to properly sized images
Improve performance drop due to properly sized images

### PR DESCRIPTION
**Pull request by andreibogdan**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #51 

Test URLs:
- Before: https://main--bitdefender--hlxsites.hlx.page/solutions/
- After: https://51-improve-performance-drop--bitdefender--hlxsites.hlx.page/solutions/

### CODE REVIEW

**Comment by andreibogdan** [comment]
Screenshot 2023-07-30 at 16 50 36

File: blocks/header/header.css [hunk 1]
```diff
@@ -182,7 +101,8 @@ header {
 }
 
 .login-modal {
-  display: flex;
+  /* display: flex; */
```
Comment (lansami) [inline blocks/header/header.css:104]: I think this should be removed

File: blocks/header/header.css [hunk 2]
```diff
@@ -680,21 +347,45 @@ nav {
...
 .options-wrapper.show{
   width: 100vw;
-  height: 65vh;
+  height: auto;
   box-shadow: 0 10px 10px #0003;
   z-index: 3;
   border-top: 1px solid #cecccc;
-  visibility: visible;
+
+  /* visibility: visible; */
```
Comment (lansami) [inline blocks/header/header.css:386]: I think this comment should be removed also

File: blocks/header/header.css [hunk 3]
```diff
@@ -736,4 +426,434 @@ nav {
...
+
+  .menu-wrapper {
+    display: none;
+  }
+
+  .nav-wrapper {
+    width: 100%;
+    background-color: var(--text-color);
+
+    /* border-bottom: 2px solid #3c3c3c;; */
```
Comment (lansami) [inline blocks/header/header.css:473]: I think this should also be removed

File: blocks/header/header.css [hunk 4]
```diff
@@ -736,4 +426,434 @@ nav {
...
+    white-space: break-spaces;
+    text-align: left;
+  }
+
+  .mega-menu > div:first-child h2 {
+    grid-column: span 3; 
+  }
+  
+  .mega-menu div p {
+    /* line-height: 0.5; */
```
Comment (lansami) [inline blocks/header/header.css:746]: comment should be removed

File: blocks/header/header.js [hunk 5]
```diff
@@ -130,9 +130,9 @@ function renderDesktopHeader(block, nav) {
     const navParagraphs = navSections.querySelectorAll('p');
     const lastNavParagraph = navParagraphs[navParagraphs.length - 1];
     navParagraphs.forEach((navParagraph) => {
-      const divider = document.createElement('div');
-      divider.className = 'nav-divider';
-      navSections.insertBefore(divider, navParagraph);
+      // const divider = document.createElement('div');
```
Comment (lansami) [inline blocks/header/header.js:133]: commented lines should be removed

File: blocks/header/header.js [hunk 6]
```diff
@@ -149,9 +149,10 @@ function renderDesktopHeader(block, nav) {
   if (navBrandLinks && navBrandLinks.length > 0) {
     const forHomeLink = Array.from(navBrandLinks)[0];
     if (forHomeLink) {
-      const homeButtonBorder = document.createElement('div');
-      homeButtonBorder.className = 'home-button-border';
-      forHomeLink.parentNode.appendChild(homeButtonBorder);
+      forHomeLink.classList.add('active');
+      // const homeButtonBorder = document.createElement('div');
```
Comment (lansami) [inline blocks/header/header.js:153]: commented lines should be removed

File: blocks/header/header.js [hunk 7]
```diff
@@ -177,15 +179,15 @@ function renderDesktopHeader(block, nav) {
   homeSolutions.addEventListener('mouseenter', () => {
     clearTimeout(hideTimeout);
     megaMenu.classList.add('mega-menu-show');
-    homeSolutions.classList.remove('home-solutions-link-default');
+    // homeSolutions.classList.remove('home-solutions-link-default');
```
Comment (lansami) [inline blocks/header/header.js:182]: should be removed

File: blocks/header/header.js [hunk 8]
```diff
@@ -177,15 +179,15 @@ function renderDesktopHeader(block, nav) {
...
     homeSolutions.classList.add('home-solutions-link-hover');
   });
 
   homeSolutions.addEventListener('mouseleave', () => {
     hideTimeout = setTimeout(() => {
       megaMenu.classList.remove('mega-menu-show');
       homeSolutions.classList.remove('home-solutions-link-hover');
-      homeSolutions.classList.add('home-solutions-link-default');
+      // homeSolutions.classList.add('home-solutions-link-default');
```
Comment (lansami) [inline blocks/header/header.js:190]: should be removed

File: blocks/header/header.js [hunk 9]
```diff
@@ -196,7 +198,7 @@ function renderDesktopHeader(block, nav) {
   megaMenu.addEventListener('mouseleave', () => {
     megaMenu.classList.remove('mega-menu-show');
     homeSolutions.classList.remove('home-solutions-link-hover');
-    homeSolutions.classList.add('home-solutions-link-default');
+    // homeSolutions.classList.add('home-solutions-link-default');
```
Comment (lansami) [inline blocks/header/header.js:201]: should be removed

File: blocks/header/header.js [hunk 10]
```diff
@@ -381,7 +389,18 @@ export default async function decorate(block) {
...
+
+    thirdChild.classList.add('logo');
+
+    const container = document.createElement('div');
+    container.classList.add('logo-container');
+    container.appendChild(thirdChild);
+
+    if (header.querySelector('p.home-solutions-link-default')) {
+      container.appendChild(header.querySelector('p.home-solutions-link-default'));
+      // header.removeChild(header.querySelector('p.home-solutions-link-default'));
```
Comment (lansami) [inline blocks/header/header.js:401]: commented line should be removed

File: scripts/lib-franklin.js [hunk 11]
```diff
@@ -616,16 +616,17 @@ export async function waitForLCP(lcpBlocks) {
...
-  await new Promise((resolve) => {
-    if (lcpCandidate && !lcpCandidate.complete) {
-      lcpCandidate.setAttribute('loading', 'eager');
-      lcpCandidate.addEventListener('load', resolve);
-      lcpCandidate.addEventListener('error', resolve);
-    } else {
-      resolve();
-    }
-  });
+  // on mobile the first image is not displayed
```
Comment (lansami) [inline scripts/lib-franklin.js:619]: commented lines should be removed

File: blocks/header/header.css [hunk 12]
```diff
@@ -736,4 +423,431 @@ nav {
...
+
+  .nav-brand p a {
+    margin: 0 19px;
+  }
+
+  .mega-menu div p a{
+    position: relative;
+    font-size: var(--body-font-size-xs);
+    font-weight: 400;
+    color: var(--text-color) !important;
```
Comment (rhudea) [inline blocks/header/header.css:525]: why !important is needed ?

File: blocks/hero/hero.css [hunk 13]
```diff
@@ -30,13 +30,18 @@ main .hero .hero-content > div {
...
 main .hero-container .hero-wrapper {
   padding: 0;
-  border-bottom: 32px solid #e4f2ff;
 }
 
 main .hero {
   position: relative;
   padding: 0;
   min-height: 300px;
+  border-bottom: 32px solid #e4f2ff;
```
Comment (rhudea) [inline blocks/hero/hero.css:39]: For colors, might worth defining variables either in block css or globally if other block are using that.

File: blocks/hero/hero.css [hunk 14]
```diff
@@ -389,16 +394,23 @@ main .hero.black-background ul.hero-awards li picture{
...
 }
 
 @media (min-width: 1600px) { /* large desktop */
-  :root {
-    --hero-font-size-l: 56px;
+  main .hero h1 {
+    font-size: 56px;
+  }
+
+  main .hero h4 {
```
Comment (rhudea) [inline blocks/hero/hero.css:408]: This doesn't seem necessary since it is already part of tablet styling

File: blocks/header/header.css [hunk 15]
```diff
@@ -289,29 +197,41 @@ header a.button::after {
...
+  display: none;
 }
 
-.header-wrapper {
-  position: relative;
-  z-index: 999;
+.login-buttons a p {
+  text-align: center;
+  font-size: var(--body-font-size-xs);
+  font-weight: 500;
```
Comment (rhudea) [inline blocks/header/header.css:224]: This should be defined as a global variable like `--font-weight-bold` and similar for other font-weights.

**Pull request merged**